### PR TITLE
[GH-611] Remove cardano-shell and monitoring dependency

### PR DESCRIPTION
**Pull request by ksaric**
https://github.com/input-output-hk/cardano-ledger/issues/611

### CODE REVIEW

File: cardano-ledger/src/Cardano/Chain/Epoch/Validation.hs [hunk 1]
```diff
@@ -51,58 +43,24 @@ data EpochError
 
 
 -- | Check that a single epoch's `Block`s are valid by folding over them
+-- TODO(KS): We should use contra-tracer here!
```
Comment (ksaric) [inline cardano-ledger/src/Cardano/Chain/Epoch/Validation.hs:46]: Yes, we should depend only on `contra-tracer` here like we do in https://github.com/input-output-hk/ouroboros-network so we don't have to pass in a ton of dependencies.
For now, the logging is removed, but I presume that people more familiar with `contra-tracer` can add it back in.
I haven't found other direct dependencies of this function, but it seems quite important.

**Review by ruhatch — APPROVED** [review]
LGTM!

File: cardano-ledger/src/Cardano/Chain/Epoch/Validation.hs (hunk 1 above)
Comment (CodiePP) [reply cardano-ledger/src/Cardano/Chain/Epoch/Validation.hs:46]: we track it in issue https://github.com/input-output-hk/iohk-monitoring-framework/issues/232

Comment (dcoutts) [reply cardano-ledger/src/Cardano/Chain/Epoch/Validation.hs:46]: Or this epoch file validation can perhaps be pulled out of the library and into a separate executable that is only used for CI. This validation functionality is not needed (afaik) by clients of the library. It is just there to help check that the validation rules can really validate mainnet.

We also can do this now in the consensus layer, and including the chain level validation, so we may end up moving it there.

**Review by dcoutts — APPROVED** [review]
LGTM.

**Comment by dcoutts** [comment]
CI failure appears spurious, but unclear how to fix it.

**Comment by deepfire** [comment]
Restarted the BORS merge -- the `cp: cannot create directory '/build/cardano-ledger': No such file or directory` failure was on `buildkite-packet-1`.

**Pull request merged**